### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by bekjon-me**
- Added index.html
- Added style.css
- Added linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
-# hello-microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
```
Comment (ha-manel) [inline README.md:5]: - [ ] The README file is the documentation of the project and making it professional reflects how professional your project is. Kindly edit this section to write the actual description of the project not just a placeholder text 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+- Html
+- Css
+
+## Live Demo (if available)
+
+Coming soon
+
+## Authors
+
+👤 **Author1**
```
Comment (ha-manel) [inline README.md:18]: - [ ] Kindly replace this text with your actual name 💯

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
-# hello-microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
```
Comment (ha-manel) [inline README.md:3]: - [ ] Kindly add the missing section `Getting Started` as shown in this [template](https://github.com/microverseinc/readme-template). In this section you're supposed to demonstrate how to use the project. (Hint: you can use [this ](https://github.com/ha-manel/Portfolio-Design)as an example)

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @bekjon-me 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Github flow is used correctly ✔️
* The files are created as required

## Required Changes:
_These are minor changes that need to be made. Please check the comments below_
------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @bekjon-me

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**